src/main/java/org/veupathdb/service/dsdl/Metrics.java

```diff
@@ -0,0 +1,15 @@
+package org.veupathdb.service.dsdl;
+
+import io.prometheus.client.Counter;
+
+public class Metrics {
+  private static final Counter DOWNLOADS_BY_DATASET = Counter.build()
```

Comment:
You've got to be careful using the words "dataset" vs "study".  The label you're attaching to the increment is a study ID e.g. PRISM0001-1 but I think if people are browsing prometheus/grafana and see dataset name, they will expect to see counts broken down by dataset e.g. DS_0ad509829e.  Maybe pick one?  Study is probably better, but you might want to ask someone.... don't know who needs this?

Reply:
I updated to use study to be consistent with the rest of the API's terminology